tests/test_pyvin.py

```diff
@@ -38,8 +62,8 @@ def test_vin_multi_large_batch(self):
     def test_invalid_in_multi(self):
         samples = (INVALID, TOYOTA_COROLLA)
         vins = VIN(*samples)
-        self.assertEqual(len([x for x in samples if x is not INVALID]),
-                         len(vins))
+        self.assertIsInstance(vins, DecodedVIN)
+        self.assertEqual(len([x for x in samples if x is not INVALID]), 1)
```

Comment:
I wasn't sure how to handle this. It would probably make more sense to always return a list. In this case, the len(vins) didn't work because it was returning a DecodedVin type instead of a list. Having to check the type when you get it back seems less than ideal, but that's an API change, so I didn't touch that.